config/features/sitenow_v2/config_split.patch.core.entity_form_display.node.page.default.yml

```diff
@@ -51,3 +51,8 @@ removing:
       weight: 2
     moderation_state:
       weight: 16
+    simple_sitemap:
+      weight: 10
+      region: content
+      settings: {  }
+      third_party_settings: {  }
```

Comment:
This seems like it should be part of the parent config that is being patched.

Reply:
```suggestion
```

Reply:
```suggestion
```